generators/app/templates/src/utils/contextFactory/interfaces.ts

```diff
@@ -0,0 +1,3 @@
+export interface ActionWithType {
```

Comment:
what the standard will be?, `interfaces.ts` or `types.ts` ?

Reply:
I don't think it makes a difference, I named it interfaces because it is an interface 🤷‍♂️